packages/core/chrome/core-chrome-browser-internal/src/ui/header/header_logo.tsx

```diff
@@ -99,8 +102,12 @@ export function HeaderLogo({ href, navigateToApp, loadingCount$, ...observables
         defaultMessage: 'Elastic home',
       })}
     >
-      <LoadingIndicator loadingCount$={loadingCount$!} />
-      <ElasticMark className="chrHeaderLogo__mark" aria-hidden={true} />
+      <LoadingIndicator loadingCount$={loadingCount$!} customLogo={logo} />
+      {customizedLogo ? (
+        <img src={customizedLogo} width="200" height="84" alt="custom mark" />
```

Comment:
Should we add i18n to the `alt` ?

Regarding the image size, how does this behave when uploading a bigger file than `200x84`? Do we have a crop/resize mechanism?

I wonder if we shouldn't better rely on `object-fit` css

```
// css
.logoWrapper img {
    width: 100%;
    height: 100%;
    object-fit: cover;
}

// Jsx
<div class="logoWrapper"><img src="..." alt="..."></div>
```